src/main/java/org/fcrepo/client/PostBuilder.java

```diff
@@ -115,6 +115,17 @@ public PostBuilder filename(final String filename) throws FcrepoOperationFailedE
     }
 
     /**
+     * Provide a content disposition header without a filename
+     *
+     * @param disposition the disposition value
+     * @return this builder
+    **/
+    public PostBuilder disposition(final String disposition) {
```

Comment:
@awoods OK, I thought that there were more options, but since there aren't, just accepting `null` to create the plain `attachment` variation seems like the easiest thing to do here.